pkg/apis/v1alpha1/provider.go

```diff
@@ -124,6 +124,9 @@ type BlockDeviceMapping struct {
 	DeviceName *string `json:"deviceName,omitempty"`
 	// EBS contains parameters used to automatically set up EBS volumes when an instance is launched.
 	EBS *BlockDevice `json:"ebs,omitempty"`
+	// RootVolume is a flag indicating if this device is mounted as kubelet root dir. You can
+	// configure at most one root volume in BlockDeviceMappings.
+	RootVolume *bool `json:"rootVolume,omitempty"`
```

Comment:
```suggestion
	RootVolume bool `json:"rootVolume,omitempty"`
```

How about just a bool since `false` by default seems to be what we would want here